cppcheck.conf

```diff
@@ -28,6 +28,9 @@ constParameter:daisi/src/path_planning/message/misc/reached_goal.h
 constParameter:daisi/src/path_planning/message/handover_message.h
 constParameter:daisi/src/path_planning/consensus/paxos/message/response_message.h
 
+# initiatorList cannot be used to assign class variable
```

Comment:
``amr_find_result_`` could be initialized in the member initializer list but that would make the code hard to read. Maybe adjust the comment :wink: